201-vmss-master-slave-customscript/README.md

```diff
@@ -8,7 +8,73 @@
 </a>
 
 ## Description
-This template allows you to deploy a VM Scale Set of Linux VMs and create a new virtual network at the same time. These VMs have a custom script extension for customization and are behind a load balancer with NAT rules for rdp connections. This allows to specify the master node number and data node number, adapt to any master / slave architecture
+This template allows you to deploy a VM Scale Set of Linux VMs and create a new virtual network at the same time. These VMs have a custom script extension for customization and are behind a load balancer with NAT rules for SSH connections. This allows to specify the master node number and data node number, adapt to any master / slave architecture
 
+## Using new features 
 
+To enable Accelerated Networking feature ( SR-IOV ) which is a free feature, using the following example : 
 
+In network profile of VMSS , set "enableAcceleratedNetworking" to true,
```

Comment:
remove the code snippets from the readme and refer to documentation or the template itself